### ISSUE TITLE
Add example with multiple utm time zones for georeferenced point clouds with cesium

### ISSUE THREAD

**Issue by nikste** [potree/potree#1171]
It seems in the examples only one pointcloud is loaded.
For multiple pointclouds you can use multiple calls to `Potree.loadPointCloud`.

However if two point clouds have a different reference utm time zone, one of them will be displayed in a wrong area.
Judging from the posts in this issue: https://github.com/potree/PotreeConverter/issues/501
and some gitter conversation the canoncial way seems to be to use `las2las` from lastools to change the reference system.
It seems this can lead to inaccuracies, however, if timezones are far apart.

I'm pretty new to potree, so is it in principle possible to load pointclouds with different utm timezones?
If so it would be great to have an example (In case the complexity is not too high i could also provide some with some guidance).

**Comment by itxiaoe** [comment]
hi, I have the same question. Have you made any progress？

**Comment by nikste** [comment]
no, so far i only used `las2las` to convert the two pointclouds to the same timezone

**Comment by itxiaoe** [comment]
well, I'm trying to use `pointcloud.translateX` move pointcloud
